synapse/storage/event_push_actions.py

```diff
@@ -223,6 +223,7 @@ def get_after_receipt(txn):
                 "   AND ep.user_id = ?"
                 "   AND ep.stream_ordering > ?"
                 "   AND ep.stream_ordering <= ?"
+                "   AND ep.notif = 1"
```

Comment:
This is probably more erik's territory but just to note that this might need some care to make sure the db does a sensible thing as this table is generally massive.